include/LArNDGeomSimple.h

```diff
@@ -0,0 +1,214 @@
+/**
+ *  @file   LArReco/include/LArNDGeomSimple.h
+ *
+ *  @brief  Header file for storing geometry information for ND LAr
+ *
+ *  $Log: $
+ */
+#ifndef PANDORA_LAR_ND_GEOM_SIMPLE_H
+#define PANDORA_LAR_ND_GEOM_SIMPLE_H 1
+
+#include "Pandora/PandoraInputTypes.h"
+
+namespace lar_nd_reco
+{
+
+class LArNDTPCSimple
+{
+public:
+    /**
+     *  @brief  default constructor
+     */
+    LArNDTPCSimple();
```

Comment:
We should remove the default constructor, since we only create valid TPC objects in `LArNDGeomSimple::AddTPC()` using the other constructor.

Reply:
`std::map` seems to want the default constructor so leaving as is.